engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -50,11 +52,14 @@ class EngineJobExecutionActor(replyTo: ActorRef,
                               jobTokenDispenserActor: ActorRef,
                               backendSingletonActor: Option[ActorRef],
                               backendName: String,
-                              callCachingMode: CallCachingMode) extends LoggingFSM[EngineJobExecutionActorState, EJEAData] with WorkflowLogging with CallMetadataHelper {
+                              callCachingMode: CallCachingMode) extends LoggingFSM[EngineJobExecutionActorState, EJEAData] 
+  with WorkflowLogging with CallMetadataHelper with JobInstrumentation {
 
   override val workflowIdForLogging = workflowDescriptor.id
   override val workflowIdForCallMetadata = workflowDescriptor.id
 
+  val jobStartTime = System.currentTimeMillis()
```

Comment:
Should this be when we get the execution token instead ?

Reply:
I think "job start time" is always going to be ambiguous. You could probably either:
- Accept it and make sure it's either written down or easy to work out
- Rename it to be verbose but unambiguous
- Quietly hope that nobody notices

Reply:
For the granularity of a dashboard i don't think we need to call in the atomic clocks